pkgs/by-name/bi/bitwarden-desktop/package.nix

```diff
@@ -195,6 +196,12 @@ in buildNpmPackage rec {
     })
   ];
 
+  passthru = {
+    updateScript = nix-update-script {
+      extraArgs = [ "--commit" "--version=stable" "--version-regex=^desktop-v(.*)$" ];
```

Comment:
Does this work for @r-ryantm or does the bot expect the update script not to commit anything?